ml-agents-envs/mlagents_envs/side_channel/environment_parameters_channel.py

```diff
@@ -35,3 +42,60 @@ def set_float_parameter(self, key: str, value: float) -> None:
         msg.write_int32(self.EnvironmentDataTypes.FLOAT)
         msg.write_float32(value)
         super().queue_message_to_send(msg)
+
+    def set_uniform_sampler_parameters(
+        self, key: str, min_value: float, max_value: float, seed: int
+    ) -> None:
+        """
+        Sets a uniform environment parameter sampler.
+        :param key: The string identifier of the parameter.
+        :param min_value: The minimum of the sampling distribution.
+        :param max_value: The maximum of the sampling distribution.
+        :param seed: The random seed to initialize the sampler.
+        """
+        msg = OutgoingMessage()
+        msg.write_string(key)
+        msg.write_int32(self.EnvironmentDataTypes.SAMPLER)
+        msg.write_int32(seed)
+        msg.write_int32(self.SamplerTypes.UNIFORM)
+        msg.write_float32(min_value)
+        msg.write_float32(max_value)
+        super().queue_message_to_send(msg)
+
+    def set_gaussian_sampler_parameters(
+        self, key: str, mean: float, st_dev: float, seed: int
+    ) -> None:
+        """
+        Sets a gaussian environment parameter sampler.
+        :param key: The string identifier of the parameter.
+        :param mean: The mean of the sampling distribution.
+        :param st_dev: The standard deviation of the sampling distribution.
+        :param seed: The random seed to initialize the sampler.
+        """
+        msg = OutgoingMessage()
+        msg.write_string(key)
+        msg.write_int32(self.EnvironmentDataTypes.SAMPLER)
+        msg.write_int32(seed)
+        msg.write_int32(self.SamplerTypes.GAUSSIAN)
+        msg.write_float32(mean)
+        msg.write_float32(st_dev)
+        super().queue_message_to_send(msg)
+
+    def set_multirangeuniform_sampler_parameters(
+        self, key: str, intervals: List[float], seed: int
+    ) -> None:
+        """
+        Sets a gaussian environment parameter sampler.
+        :param key: The string identifier of the parameter.
+        :param intervals: The min and max that define each uniform distribution.
+        :param seed: The random seed to initialize the sampler.
+        """
+        msg = OutgoingMessage()
+        msg.write_string(key)
+        msg.write_int32(self.EnvironmentDataTypes.SAMPLER)
+        msg.write_int32(seed)
+        msg.write_int32(self.SamplerTypes.MULTIRANGEUNIFORM)
+        msg.write_int32(len(intervals))
+        for value in intervals:
+            msg.write_float32(value)
```

Comment:
```suggestion
        msg.write_float32_list(intervals)
```
There's already a helper function for this